test/unit/nunjuckUtils.test.js

```diff
@@ -0,0 +1,30 @@
+const nunjucks = require('nunjucks');
+const nunjuckUtils = require('../../src/lib/markbind/nunjuckUtils');
+
+test('Escaping nunjucks raw tags', () => {
+  const escapedString = 'This is a content with escaped data {%raw%} CONTENT {%endraw%}';
+  const escapedContent = nunjuckUtils.renderEscaped(nunjucks, escapedString);
+
+  expect(escapedContent).toBe(escapedString);
+});
+
+test('Escaping nunjucks with new lines', () => {
+  const escapedString = 'This is a content with escaped data\n {%raw%} \nCONTENT\n {%endraw%}';
+  const escapedContent = nunjuckUtils.renderEscaped(nunjucks, escapedString);
+
+  expect(escapedContent).toBe(escapedString);
+});
+
+test('Escaping multiple nunjucks raw tags', () => {
+  const escapedString = 'Multiple escapes: {%raw%} first {%endraw%} {%raw%} second {%endraw%}';
+  const escapedContent = nunjuckUtils.renderEscaped(nunjucks, escapedString);
+
+  expect(escapedContent).toBe(escapedString);
+});
+
```

Comment:
Do you think this nested escapes are a case we need to account for?

```
test('Escaping nested nunjucks raw tags', () => {
  const escapedString = 'This is a content with nested escaped data {%raw%} {%raw%} CONTENT {%endraw%} {%endraw%}';
  const escapedContent = nunjuckUtils.renderEscaped(nunjucks, escapedString);
  expect(escapedContent).toBe(escapedString);
});
```

I can imagine this might be useful for documenting how to use to use the escape tags in the first place

Reply:
Nesting with stuff in between doesn't seem to work correctly 😅 
```
test('Escaping nested nunjucks raw tags', () => {
  const escapedString = 'Nested escaped data: {% raw %}BEFORE{% raw %} CONTENT {% endraw %}AFTER{% endraw %}';
  const escapedContent = nunjuckUtils.renderEscaped(nunjucks, escapedString);
  expect(escapedContent).toBe(escapedString);
});
```
```
Expected: "Nested escaped data: {% raw %}BEFORE{% raw %} CONTENT {% endraw %}AFTER{% endraw %}"
Received: "Nested escaped data: {% raw %}BEFORE{% raw %} CONTENT {% endraw %}{% endraw %}AFTER"
```

Reply:
It feels like as long as we go with the opentag/closetag approach, we would always run into this issue:
`$OPEN$ $CLOSE$ $CLOSE$` <- how do we match the `$OPEN` to the right `$CLOSE`?

Do you think having a special tag that escapes the next character would help? This would "break" the syntax so the original special tag wont be treated as such.